data/presets/club/scout.json

```
@@ -0,0 +1,21 @@
+{
+    "icon": "fas-handshake",
+    "fields": [
+        "name",
+        "scout",
+        "operator",
+        "brand"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "club": "scout"
+    },
+    "terms": [
+        "pathfinder",
+        "guide"
```

please sort the `terms` alphabetically.

also, we could add _scouting_ as a term:

```suggestion
        "guide",
        "pathfinder",
        "scouting"
```